src/styles/components/_header.scss

```diff
@@ -251,18 +251,29 @@
     }
   }
 
-  // expand search block for middle resolutions
-  @include mq($min-width: $vw-large + 1, $max-width: $vw-huge) {
+  // expand search block for specific resolutions
+  @include mq($min-width: $vw-huge + 1, $max-width: 1300px) {
+    .search.header__search {
+      width: 20px;
```

Comment:
We don't see the input at all in this resolution (when not expanded).

Reply:
That's the point^^
This input is ridiculously small at this resolution, I prefer to hide it and only display it when clicking on the search icon.